packages/tools/client-debugger/client-debugger-view/src/components/WaitingLabels.tsx

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Type of {@link waitingLabels}.
+ */
+export type WaitingLabelsType = Record<string, string>;
+
+/**
+ * Record containing labels for {@link Waiting} component.
+ */
+export const waitingLabels: WaitingLabelsType = {
+	containerError: "Waiting for container DDS data.",
```

Comment:
This isn't really an error. This is just a label we use while we're waiting for a data request to be returned.